draft-ietf-gnap-core-protocol.md

```diff
@@ -5774,7 +5792,7 @@ Cache-Control: no-store
 
 
 The client instance saves the response and redirects the user to the
-interaction_url by sending the following HTTP message to the user's
+interaction_uri by sending the following HTTP message to the user's
```

Comment:
```suggestion
redirect interaction start URI by sending the following HTTP message to the user's
```

Reply:
Reverted, not sure what the suggested change was about